context/app/static/js/components/Detail/DetailLayout/DetailLayout.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import NoticeAlert from '../../NoticeAlert';
 import TableOfContents from '../TableOfContents';
-import { FlexColumn, FlexRow } from './style';
+import { Content, FlexRow } from './style';
 import { capitalizeString } from '../../../helpers/functions';
```

Comment:
use the helpers alias? Maybe do a global search to clear this up if there are other instances.

Reply:
Sure, this was just a relic from before I added the helpers alias.